wasm/bindings_wasm.d.ts

```diff
@@ -31,10 +31,10 @@ export interface InitOutput {
   readonly xxhashBase36: (a: number, b: number) => void;
   readonly xxhashBase16: (a: number, b: number) => void;
   readonly __wbindgen_add_to_stack_pointer: (a: number) => number;
-  readonly __wbindgen_malloc: (a: number, b: number) => number;
-  readonly __wbindgen_realloc: (a: number, b: number, c: number, d: number) => number;
-  readonly __wbindgen_free: (a: number, b: number, c: number) => void;
-  readonly __wbindgen_exn_store: (a: number) => void;
+  readonly __wbindgen_export_0: (a: number, b: number) => number;
+  readonly __wbindgen_export_1: (a: number, b: number, c: number, d: number) => number;
+  readonly __wbindgen_export_2: (a: number, b: number, c: number) => void;
+  readonly __wbindgen_export_3: (a: number) => void;
```

Comment:
Where do these changes come from? I do not have them locally when building. What platform are you building on?